api/models.go

```diff
@@ -85,14 +85,15 @@ type UpdateAppRequest struct {
 }
 
 type CreateAppRequest struct {
-	Name          string   `json:"name"`
-	Pubkey        string   `json:"pubkey"`
-	MaxAmountSat  uint64   `json:"maxAmount"`
-	BudgetRenewal string   `json:"budgetRenewal"`
-	ExpiresAt     string   `json:"expiresAt"`
-	Scopes        []string `json:"scopes"`
-	ReturnTo      string   `json:"returnTo"`
-	Isolated      bool     `json:"isolated"`
+	Name             string   `json:"name"`
+	Pubkey           string   `json:"pubkey"`
+	MaxAmountSat     uint64   `json:"maxAmount"`
+	BudgetRenewal    string   `json:"budgetRenewal"`
+	ExpiresAt        string   `json:"expiresAt"`
+	Scopes           []string `json:"scopes"`
+	ReturnTo         string   `json:"returnTo"`
+	Isolated         bool     `json:"isolated"`
+	LightningAddress string
```

Comment:
Interesting you also allow the lightning address to be passed from the frontend, is there a usecase for this?

Reply:
The easiest way was not to change CreateApp args.

It doesn't have json field so no. Also in http service it's always set to empty string first. https://github.com/getAlby/hub/pull/560/files#diff-3d9fb3ef05f4794887b3f5b899cf421382e8a466f4b33692389ed2af9ad31728R851

Reply:
Ok. Unless there is a clear usecase I would not add this property to the request payload